engine/src/test/java/org/pentaho/di/trans/steps/transexecutor/TransExecutorHelperTest.java

```diff
@@ -0,0 +1,125 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+
+package org.pentaho.di.trans.steps.transexecutor;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.KettleLogStore;
+import org.pentaho.di.core.parameters.UnknownParamException;
+import org.pentaho.di.repository.Repository;
+import org.pentaho.di.trans.TransMeta;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+import static org.pentaho.di.trans.step.StepHelperInterface.ACTION_STATUS;
+import static org.pentaho.di.trans.step.StepHelperInterface.FAILURE_METHOD_NOT_RESPONSE;
+import static org.pentaho.di.trans.step.StepHelperInterface.FAILURE_RESPONSE;
+
+public class TransExecutorHelperTest {
+
+  TransExecutorMeta transExecutorMeta;
+  TransMeta transMeta;
+
+  TransExecutorHelper underTest;
+
+  @BeforeClass
+  public static void initKettle() {
+    KettleLogStore.init();
```

Comment:
No change necessary
I assume for testing you are calling  `KettleLogStore.init()` due to 

```
  protected BaseStepHelper() {
    this.log = KettleLogStore.getLogChannelInterfaceFactory().create( this );
  }
```

You can use mockito's static mock to mock `KettleLogStore.getLogChannelInterfaceFactory()`,

I doubt you are actually using it

These Kettle***,init() functions make unit tests slow, and honestly turn these "unit tests: into more of integation tests

references:
- https://www.baeldung.com/mockito-mock-static-methods